modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/PlaybackManager.kt

```diff
@@ -1691,34 +1701,6 @@ open class PlaybackManager @Inject constructor(
         }
     }
 
-    /**
-     * Check we have the latest episode url in the background.
-     */
-    private fun updateEpisodeUrl(episode: PodcastEpisode) {
-        if (episode.isDownloaded) {
-            return
-        }
-
-        // TODO
-//        val updateEpisodeTask = UpdateEpisodeTask(podcastCacheServerManager, episodeManager)
-//        val episodeSingle = updateEpisodeTask.download(episode.podcastUuid, episode.uuid)
-//        val disposable = episodeSingle
-//                .subscribeOn(Schedulers.io())
-//                .subscribe(
-//                        { episodeUpdated ->
-//                            val currentEpisode = getCurrentEpisode()
-//                            if (currentEpisode != null &&
-//                                    currentEpisode.uuid == episodeUpdated.uuid &&
-//                                    currentEpisode.downloadUrl != episodeUpdated.downloadUrl) {
-//                                upNextQueue.reloadCurrentEpisode()
-//                                forcePlayerSwitch = true
-//                            }
-//                        },
-//                        { throwable -> Timber.e(throwable, "Failed to update episode url") }
-//                )
-//        disposables.add(disposable)
```

Comment:
You don't by chance remember why we removed this do you @geekygecko ? This is from around 5 years ago, so I doubt you'll have any recollection, but I just wanted to make sure there wasn't anything we should be watching out for in this PR.

Reply:
Sorry, I can't remember why we removed it. But it's not called in iOS either now so there must have been a reason. I wonder if it was too slow calling it before we played the episode.

Reply:
No worries.

> it's not called in iOS either now so there must have been a reason

iOS does seem to update the URL before playing an episode [here](https://github.com/Automattic/pocket-casts-ios/blob/trunk/podcasts/PlaybackActionHelper.swift#L93) I believe.